### PR TITLE
feat: use Sorensen prevent default down option and configurable poison key

### PR DESCRIPTION
**Pull request by KvelaGorrrrnio**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix

* **What is the current behavior?** (You can also link to an open issue here)
Shortcuts that are bound on keyUp and conflicts with browser shortcuts doesn't prevent default behaviour.

* **What is the new behavior (if this is a feature change)?**
Shortcuts prevent default behaviour on keyDown.

* **Other information**:
The code that prevents default behaviour for `Ctrl+Digit` and function keys has not been removed, since they might be preferred prevented, even when no action triggers have been defined for that combo.

**Status**

- [X] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [X] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/client/lib/triggers/TriggersHandler.tsx [hunk 1]
```diff
@@ -304,11 +306,15 @@ export const TriggersHandler: React.FC<IProps> = function TriggersHandler(
 		]
 		const systemActionKeys = ['Enter', 'NumpadEnter', 'Tab']
 
+		const poisonKey: string | null = Meteor.settings.poisonKey
```
Comment (jstarpl) [inline meteor/client/lib/triggers/TriggersHandler.tsx:309]: In the way this is currently set up, this needs to import `Settings` from `meteor\lib\Settings.ts` and use `Settings.poisonKey` as that's where the "defaulted" version of settings lives.

**Review by jstarpl — CHANGES_REQUESTED** [review]
Looks good, apart from the default `poisonKey` which wouldn't work, as the code is now.

File: meteor/client/lib/triggers/TriggersHandler.tsx (hunk 1 above)
Comment (KvelaGorrrrnio) [reply meteor/client/lib/triggers/TriggersHandler.tsx]: @jstarpl Thanks for the catch. It is fixed now.

**Review by jstarpl — APPROVED** [review]
Looks good now!

**Pull request merged**